src/store/reducers/describe.ts

```diff
@@ -13,7 +17,10 @@ const describe = function z(state = {loading: false, wasLoaded: false, data: {}}
         }
         case FETCH_DESCRIBE.SUCCESS: {
             const newData = JSON.parse(JSON.stringify(state.data));
-            newData[action.data.Path] = action.data;
+
+            // In  case of successful fetch data should always has some Path, so we could cast type here
```

Comment:
should _have_

What exactly enforces this statement? Looks like a hopeful assumption :)